src/app/content/components/Page.tsx

```diff
@@ -30,6 +30,10 @@ import { toRelativeUrl } from '../utils/urlUtils';
 import { contentTextWidth } from './constants';
 import allImagesLoaded from './utils/allImagesLoaded';
 
+if (typeof(document) !== 'undefined') {
+  import(/* webpackChunkName: "NodeList.forEach" */ 'mdn-polyfills/NodeList.prototype.forEach');
+}
```

Comment:
Useful to add a comment re what this is doing? (might be hard for future readers or current me to figure out :-)